### PR TITLE
Add skipable rake config

### PR DESCRIPTION
**Pull request by carl-printreleaf**
When running asset precompile for rails, rails performance gem tries to connect to redis. This change gives the option to exclude certain rake tasks from performance tracking.

Config example:
```ruby
    config.skipable_rake_tasks = ['assets:precompile']
```

### CODE REVIEW

File: test/rake_ext_test.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class RakeExtTest < ActiveSupport::TestCase
+  test "can exclude rake tasks" do
```
Comment (carl-printreleaf) [inline test/rake_ext_test.rb:4]: Can't get this test to work, RakeExt is causing an infinite loop.
```ruby
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
rails_performance/lib/rails_performance/gems/rake_ext.rb:11:in `invoke_with_rails_performance'
rails_performance/lib/rails_performance/gems/rake_ext.rb:31:in `invoke'
```

File: app/views/rails_performance/stylesheets/style.css [hunk 2]
```diff
@@ -81,6 +81,9 @@
   border-bottom: 1px solid red;
 }
 
+.card-content {
+  overflow-x: scroll;
+}
```
Comment (carl-printreleaf) [inline app/views/rails_performance/stylesheets/style.css:86]: I thought this was included in my other pr that was merged. This change helped fix overflow issues on the tables in the UI.

File: test/rake_ext_test.rb (hunk 1 above)
Comment (igorkasyanchuk) [reply test/rake_ext_test.rb:4]: @carl-printreleaf can you try this approach https://gist.github.com/igorkasyanchuk/b828f483d0d132c1ca43733b101ce5e0

it works locally, but maybe you can check it too and prepare a PR

Comment (carl-printreleaf) [reply test/rake_ext_test.rb:4]: @igorkasyanchuk thanks for the reply, I'll try it out and update my pr if it works

Comment (carl-printreleaf) [reply test/rake_ext_test.rb:13]: @igorkasyanchuk your suggestion worked. I updated the pr. Let me know if you have any other comments.

**Comment by igorkasyanchuk** [comment]
rails_performance-1.0.4.gem released, thanks!

**Pull request merged**